debian/autobake-deb.sh

```diff
@@ -6,87 +6,17 @@
 # Exit immediately on any error
 set -e
 
-# On Buildbot, don't run the mysql-test-run test suite as part of build.
-# It takes a lot of time, and we will do a better test anyway in
-# Buildbot, running the test suite from installed .debs on a clean VM.
-# On Travis-CI we want to simulate the full build, including tests.
-# Also on Travis-CI it is useful not to override the DEB_BUILD_OPTIONS
-# at this stage at all.
-if [[ ! $TRAVIS ]]
-then
-  export DEB_BUILD_OPTIONS="nocheck"
-fi
-
-# Travis-CI optimizations
-if [[ $TRAVIS ]]
-then
-  # On Travis-CI, the log must stay under 4MB so make the build less verbose
-  sed -i -e '/Add support for verbose builds/,+2d' debian/rules
-
-  # Don't include test suite package on Travis-CI to make the build time shorter
-  sed '/Package: mariadb-test-data/,/^$/d' -i debian/control
-  sed '/Package: mariadb-test/,/^$/d' -i debian/control
-  sed '/Package: mariadb-plugin-tokudb/,/^$/d' -i debian/control
-  sed '/Package: mariadb-plugin-mroonga/,/^$/d' -i debian/control
-  sed '/Package: mariadb-plugin-spider/,/^$/d' -i debian/control
-  sed '/Package: mariadb-plugin-oqgraph/,/^$/d' -i debian/control
-  export MYSQL_COMPILER_LAUNCHER=ccache
-  sed 's|-DDEB|-DPLUGIN_TOKUDB=NO -DPLUGIN_MROONGA=NO -DPLUGIN_SPIDER=NO -DPLUGIN_OQGRAPH=NO -DPLUGIN_PERFSCHEMA=NO -WITH_EMBEDDED_SERVER=OFF -DDEB|' -i debian/rules
-fi
-
+export DEB_BUILD_OPTIONS="nocheck"
 
```

Comment:
Is it so that there is no need of running checks when building with autobake-deb.sh`

Reply:
Good question, I was just removing the travis condition. It does look like the autobake builders in buildbot do not run tests because of this. I don't know the reasons behind this choice.